CONTRIBUTING.md

```diff
@@ -1,17 +1,17 @@
 # Contributing
 
-We absolutely welcome any code contributions, and we hope that this
-guide will facilitate an understanding of the openapi-common code
-repository.
+Code contributions are welcome. The objective of the documentation
+for the Open API-Common libary is to facilitate an understanding
```

Comment:
I know 'Open API-Common' is consistent with other uses of Open API, but having it hyphenated in this way is just a bit difficult to parse: I just read it as Open... API-Common, and then I'm wondering what API-Common is. I think OpenAPI-Common works as a capitalization scheme, but the extra space is just a jarring.

Reply:
I see you've referred to OpenAPI-Common further down, so is this just a typo?